### PR TITLE
Update Chinese Translation

### PR DESCRIPTION
**Pull request by Sdarfeesh**

### CODE REVIEW

File: app/src/main/res/values-zh/strings.xml [hunk 1]
```diff
@@ -100,29 +185,33 @@
...
 
     <string name="torrent_plot">简介</string>
     <string name="normal_no_plot">没有找到简介</string>
     <string name="torrent_no_plot">没有找到简介</string>
 
+    <string name="show_log_cat">显示日志</string>
+
     <string name="picture_in_picture">画中画</string>
-    <string name="picture_in_picture_des">在其他应用之上的迷你播放器中继续播放</string>
+    <string name="picture_in_picture_des">>在其他应用之上的迷你播放器中继续播放</string>
```
Comment (LagradOst) [inline app/src/main/res/values-zh/strings.xml:214]: is the extra > intended?

File: app/src/main/res/values-zh/strings.xml [hunk 2]
```diff
@@ -1,36 +1,113 @@
 <!--https://newbedev.com/concatenate-multiple-strings-in-xml-->
 <resources>
+    <!-- KEYS DON'T TRANSLATE -->
```
Comment (LagradOst) [inline app/src/main/res/values-zh/strings.xml:3]: You can remove all these keys

**Review by LagradOst — APPROVED** [review]
Very good 👍

File: app/src/main/res/values-zh/strings.xml (hunk 1 above)
Comment (Sdarfeesh) [reply app/src/main/res/values-zh/strings.xml]: The extra > deleted

**Comment by Sdarfeesh** [comment]
> Looks good, ready to merge? 👍

Yes

**Pull request merged**